dev/io.openliberty.concurrent.internal.cdi/resources/io/openliberty/concurrent/internal/cdi/resources/CWWKCMessages.nlsprops

```diff
@@ -24,5 +24,33 @@
 #   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
 #   (single quote must be coded as one single quote '). 
 
-# All messages must use the range CWWCK1410 to CWWCK1499
+# All messages must use the range CWWKC1410 to CWWKC1499
 
+CWWKC1410.schedule.lacks.seconds=CWWKC1410E: The Schedule, {0}, specifies an \
```

Comment:
The Schedule annotation, {0}, specifies an empty value for the seconds at which the task runs. Specify a nonempty value for the seconds field, or omit the seconds field from the Schedule annotation. Omitting the seconds field schedules the task to run at the start of the minute by default. 

The empty array value indicates that the seconds field must be disregarded when scheduled times are computed. The task ends if the seconds field is empty. Omitting the seconds field schedules the task to run at the start of the minute by default. 

Modify the seconds field of the Schedule annotation to specify a nonempty value, or omit the seconds field from the Schedule.